tcms/core/views.py

```diff
@@ -121,12 +121,19 @@ def get_context_data(self, **kwargs):
             .distinct()
         )
 
+        tenant_name = ""
+        if hasattr(self.request, "tenant"):
+            tenant_name = self.request.tenant.schema_name
+            if self.request.tenant.organization:
+                tenant_name = self.request.tenant.organization
+
```

Comment:
Remove all of this. We don't want to have plugins interlinked with the main app too much.